docs/overview.adoc

```diff
@@ -99,6 +99,11 @@ image::{img}/apoc.index.nodes-with-score.jpg[width=600]
 | apoc.index.between(node1,'TYPE',node2,'prop:value*') YIELD rel, weight | lucene query on relationship index with the given type name bound by either or both sides (each node parameter can be null)
 | apoc.index.out(node,'TYPE','prop:value*') YIELD node, weight | lucene query on relationship index with the given type name for *outgoing* relationship of the given node, *returns end-nodes*
 | apoc.index.in(node,'TYPE','prop:value*') YIELD node, weight | lucene query on relationship index with the given type name for *incoming* relationship of the given node, *returns start-nodes*
+| apoc.index.nodes.count('Label','prop:value*') YIELD value | lucene query on node index with the given label name
```

Comment:
Aggiungere cosa ritorna come nell'esempio della riga precedente "\*returns start-nodes*":

`apoc.index.in(node,'TYPE','prop:value*') YIELD node, weight | lucene query on relationship index with the given type name for *incoming* relationship of the given node, *returns start-nodes*`